Product/Production/Services/AuditRepositoryCore/src/test/java/gov/hhs/fha/nhinc/audit/transform/AuditTransformsTest.java

```diff
@@ -135,36 +134,36 @@ protected void testGetEventIdentificationType(LogEventRequestType request, Strin
      * @param assertion
      */
     protected void testCreateActiveParticipantFromUser(LogEventRequestType request, Boolean isRequesting,
-            AssertionType assertion) {
+        AssertionType assertion) {
 
         if (isRequesting) {
             ActiveParticipant userActiveParticipant = null;
             List<ActiveParticipant> activeParticipant = request.getAuditMessage().getActiveParticipant();
             for (ActiveParticipant item : activeParticipant) {
                 // TODO: Where does this hard-coded value of "Code" come from?
                 if (item.getRoleIDCode().get(0).getCode() != null
-                        && item.getRoleIDCode().get(0).getCode().equals("Code")) {
+                    && item.getRoleIDCode().get(0).getCode().equals("Code")) {
 
                     userActiveParticipant = item;
                     break;
                 }
             }
 
             assertNotNull("userActiveParticipant is null", userActiveParticipant);
-            assertEquals("UserID mismatch", assertion.getUserInfo().getUserName(), userActiveParticipant.getUserID());
```

Comment:
Why are we loosening this test? Seems we knew the User ID before this change. Is it no longer possible to match this on the passed in assertion